utils/oscap.8

```diff
@@ -95,6 +95,11 @@ Use given CPE dictionary or language (auto-detected) for applicability checks. (
 Write XCCDF results into FILE.
 .RE
 .TP
+\fB\-\-thin-results\fR
```

Comment:
Yeah, you are right. I was using `--thin` all the time which also works so I assumed it's `--thin`.